docs/walkthrough/index.md

```diff
@@ -0,0 +1,272 @@
+
+Running the deployment command from the previous section should give you an output that includes a script similar to this one:
+
+```bash
+cat > promitor-scrape-config.yaml <<EOF
+extraScrapeConfigs: |
+  - job_name: promitor-agent-scraper
+    metrics_path: /metrics
+    static_configs:
+      - targets:
+        - promitor-agent-scraper.default.svc.cluster.local:80
+EOF
+helm install stable/prometheus -f promitor-scrape-config.yaml
+```
+
+You can see this output again at any time by running `helm status promitor-agent-scraper`.
+
+Running these commands will create a Prometheus scraping configuration file in your current directory and deploy Prometheus to your cluster with that scraping configuration in addition to the default.
+
+## Add load to the queue
+
+Now we'll use Service Bus Explorer to add load to our Service Bus queue so there are meaningful metrics for Promitor to pick up.
+
+In Service Bus Explorer, you can connect to your namespace & queue using a connection string. From there, right clicking on the queue in the side-bar should give you an option to 'Send Message' - from there, use the 'Sender' tab of that window to send bulk messages. Remember how many you send - you should see that number in the Promitor & Prometheus output.
+
+## See Promitor & Prometheus output via port-forwarding
+
+Going back to your cluster, you should be able to see all Promitor & Prometheus pods up and running with `kubectl get pods`.
+
+You can also see the services which provide a stable endpoint at which to reach the pods by running `kubectl get services`.
+
+This should give you a list with output similar to:
+
+| NAME | TYPE | CLUSTER-IP | EXTERNAL-IP | PORT(S) |
+| ---- | ---- | ---------- | ----------- | ------- |
+| promitor-agent-scraper | ClusterIP | 10.0.#.# | \<none\> | 80/TCP |
+| \<prometheus-release-name\>-prometheus-server | ClusterIP | 10.0.#.# | \<none\> | 80/TCP |
+
+Next to that, it should also list other services deployed by Prometheus.
+
+Let's first look at the Promitor output!
+
+Run `kubectl port-forward svc/promitor-agent-scraper 8080:80` and check http://localhost:8080/metrics. You should see some information about your queue:
+
+```bash
+# TODO sample output
+```
+
+We can also look at the Prometheus server and check that it's pulling in metrics from Promitor.
+
+Cancel the previous port-forward command and run `kubectl port-forward svc/<prometheus-release-name>-prometheus-server 8080:80`.
+
+Now, if you check http://localhost:8080, you should be able to enter Prometheus queries.
+
+Query `demo_queue_size` and as long as all your pods are up and running and both Promitor and Prometheus have scraped metrics at least once, you should see a value that matches the number of messages in your queue.
+
+## Set up alerts with Prometheus
```

Comment:
Maybe move this after creating the dashboard?

Visualizing metric before alerting on it feels more natural to me, but I might be wrong

Reply:
Sure thing, will do